### ISSUE TITLE
user password change

### ISSUE THREAD

**Issue by lymanlai** [deployd/deployd#209]
How to test a username,password match using dpd.users.login?? while I using it in a event resource, it get the result of '{"message":"resource not found","status":404}'

the case is this:
1. user send currentPassword and new password to my custom event resource: change-password
2. I am trying to using dpd.users.login({username:username, password:currentPassword},function(result,err){});
3. but then it get the err: '{"message":"resource not found","status":404}'

is that the dpd.users.login method can not used in the internal scope??

**Comment by ginuim** [comment]
@ritch I agree with robrusher, Password reset is a basic feature that should be part of the core for Deployd,Every application that has users and passwords has a reset feature.
